lib/chef/guard_interpreter/resource_guard_interpreter.rb

```diff
@@ -20,19 +20,28 @@
 
 class Chef
   class GuardInterpreter
-    class ResourceGuardInterpreter < DefaultGuardInterpreter
```

Comment:
No behavior was inherited other than `initialize` which made this more convoluted.